index.html

```diff
@@ -200,11 +200,40 @@ <h5 class="pt-4 pb-2 lh-lg"><span id="qtLabel">Choose from some of ESP's pre-bui
                                 </div>
                             </div>
                         </div>
+                        <div class="field-container" id ="developKitsContainer" style="display: none">
+                            <label>
+                                ESP Develop Kits
+                            </label>
+                            <div class="field">
+                                <div id="developKits">
+                                    <div class="form-check-inline">
+                                        <label class="form-check-label" for="radio3">ESP32-S3-DevKits&nbsp;
+                                            <input type="radio" checked class="form-check-input" name="developKitsType" id="radio3" value="esp32_matter_all-clusters-app_te6_merged.bin">
```

Comment:
Why are these added here matter_all_clusters-app?

Reply:
I forgot to make the changes.

Reply:
Modify it to not assign values by default.